src/wazuh_db/wdb.h

```diff
@@ -1316,6 +1330,16 @@ int wdb_parse_global_sync_agent_info_set(wdb_t * wdb, char * input, char * outpu
  */
 int wdb_parse_global_get_agents_by_keepalive(wdb_t* wdb, char* input, char* output);
 
+/**
+ * @brief Function to parse the disconnect-agents command data.
+ *
+ * @param [in] wdb The global struct database.
+ * @param [in] input String with the time threshold before which consider an agent as disconnected.
+ * @param [out] output Response of the command in JSON format with the list of agents that were set as disconnected.
+ * @return 0 Success: response contains the value. -1 On error: invalid DB query syntax.
```

Comment:
We can use
" 
```
* @return 0 Success: response contains "ok".
*        -1 On error: response contains "err" and an error description.
```
"
Most of parse method does and is easier to read.

Reply:
Fixed